src/components/editor/PropertyTemplateOutline.jsx

```diff
@@ -16,40 +16,37 @@ import PropTypes from 'prop-types'
 import shortid from 'shortid'
 const _ = require('lodash')
 
-export const valueTemplateRefTest = (property) => {
-  return Boolean(property.valueConstraint != null &&
-    property.valueConstraint.valueTemplateRefs != null &&
-    property.valueConstraint.valueTemplateRefs.length > 0)
+export const hasValueTemplateRef = (property) => {
+  return Boolean(property.valueConstraint && property.valueConstraint.valueTemplateRefs)
```

Comment:
this approach looking for non-null values is shorter and more ruby-like.  Perhaps we need to keep the `.length` check tho??

Reply:
Yeah, but this is functional javascript code and adding an extra Boolean class here is not necessary.